src/Modules/ExteriorPowers/SubQuo.jl

```diff
@@ -14,7 +14,9 @@ function exterior_power(M::SubquoModule, p::Int; cached::Bool=true)
   else
     C = presentation(M)
     phi = map(C, 1)
-    codomain(phi).S = Symbol.(["$e" for e in gens(M)])
+    codomain(phi).S = function _get_symbol()
+      return Symbol.(["$e" for e in gens(M)])
```

Comment:
I think this allocates a `Vector{String}` and then a `Vector{Symbol}` which you can avoid:
```suggestion
      return [Symbol"$e") for e in gens(M)]
```